tests/test_func.py

```diff
@@ -0,0 +1,145 @@
+import deeplink
```

Comment:
This test file misses the case where the target directory is inside the source one.